.github/workflows/main.yml

```diff
@@ -1,4 +1,4 @@
-name: taisei-build-tests
+name: Taisei Test Build (CI)
```

Comment:
Minor nitpick, but I'm not a fan of the verbose names. It looks overloaded, and one of the entries even obscures the build time:
![image](https://user-images.githubusercontent.com/609016/119096571-6ddd4380-ba1c-11eb-9215-562a2430e01f.png)

* It is obvious that we're building Taisei, so that part can be dropped
* Likewise, the "(CI)" offers no useful information
* Since we have "Test Build" in the suite name, we don't need to restate "Test" in each run
* Emscripten builds always use WebGL, so we don't need to state that either
* "emsdk" can be dropped, just the emscripten version in parentheses is enough
    * On that note, I think "experimental" should be replaced with "latest" since that's the tag they're using
    * Not sure if possible, but it'd be nice to have the actual version (i.e. the value of `EM_VERSION` from `env`) instead of "stable"

Reply:
Done. I'd actually moved to this short form in the release pipeline branch, but forgot to port those changes over to these tests. 

As for the actual version being in the name, I tried a bunch of ways of getting the version number into the actual name of the test, but unfortunately it's not possible throw any `env` vars into it for whatever reason. 

One way of doing it would be to use a `strategy: matrix:` which would allow us to access `{{ matrix.* }}` variables (allowing us to define `matrix.emsdk`, for example), but while that *is* a lot more powerful, it's also a bit overkill for what we've got going on here, and would be a whole big change all on its own. Maybe if we decide we really want to test against 6 different versions of Ubuntu (which is pretty much what `matrix` is intended for), then it'd be worth it.